compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeLayer.desktop.kt

```diff
@@ -235,6 +236,12 @@ internal class ComposeLayer {
         isDisposed = true
     }
 
+    var compositionLocalContext: CompositionLocalContext?
+        get() = scene.compositionLocalContext
```

Comment:
Maybe https://kotlinlang.org/docs/delegated-properties.html#delegating-to-another-property

Reply:
Wow, I was waiting for this feature for years :). Fixed.